protocol/x/revshare/keeper/revshare.go

```diff
@@ -119,6 +119,9 @@ func (k Keeper) SetUnconditionalRevShareConfigParams(ctx sdk.Context, config typ
 	store.Set([]byte(types.UnconditionalRevShareConfigKey), unconditionalRevShareConfigBytes)
 }
 
+// ValidateRevShareSafety checks if the total rev share ppm is less than 1 million.
+// i.e the total percentage of fees that can be shared is less than 100%
```

Comment:
By "in math formula", I was thinking sth along the lines of:

`highest_affiliate_taker_share + sum(unconditional_rev_shares) + market_mapper_rev_share < 100%`

Generally we explicitly write these formula out in protocol comments for readability purpose. However, reading above carefully the formula isn't sound since it's adding taker rev share and net rev share. We'll need to add HigherMakerFee and lowestTakerFee to this formula. Let's just call out the formula is an estimate on total rev share for now.